### PR TITLE
codeowners: update new owners

### PR DESCRIPTION
**Pull request by mythi**
Signed-off-by: Mikko Ylinen <mikko.ylinen@intel.com>

### CODE REVIEW

**Review by rojkov — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,36 +2,39 @@
...
 
 # Binaries
-/cmd/               @rojkov @bart0sh
-/cmd/fpga_tool/     @bart0sh @kad @rojkov
-/cmd/gpu_plugin/    @rojkov @bart0sh @uniemimu
-/cmd/qat_plugin/    @rojkov @mythi
-/cmd/vpu_plugin/    @rojkov @mythi
+/cmd/               @mythi @bart0sh
+/cmd/fpga_tool/     @bart0sh @kad @mythi
+/cmd/gpu_plugin/    @tkatila @bart0sh @uniemimu
```
Comment (tkatila) [inline .github/CODEOWNERS:17]: What about gpu_nfdhook ?

**Review by bart0sh — DISMISSED** [review]
/lgtm

File: .github/CODEOWNERS (hunk 1 above)
Comment (bart0sh) [reply .github/CODEOWNERS:17]: @tkatila any proposals? as far as I understand cmd/ owners have access to everything under cmd/, so @mythi and @bart0sh are owners of gpu_nfdhook by default.

Comment (mythi) [reply .github/CODEOWNERS:17]: I added gpu_nfdhook same as gpu_plugin.

**Review by bart0sh — APPROVED** [review]

**Pull request merged**